its/ruling/src/test/resources/jdk6/squid-S4274.json

```diff
@@ -0,0 +1,31 @@
+{
+'jdk6:java/awt/AWTKeyStroke.java':[
+857,
```

Comment:
I would prefer to have also the 2 next statements raising an issue. It would make more sense to me to raise issues as soon as there is an usage, even if it's as parameter of method invocation.